### PR TITLE
enhancement/1538 - Update KDCA

### LINKED ISSUE #1538: Fix KDCA erroneous coordinates and update procedures
The BKW VOR seems to be _very_ off
This is a good opportunity to update this airport to current AIRAC. Very few changes to some procedures and one new SID. Traffic could also be updated.

### LINKED ISSUE #1542: KDCA erroneous STARs = often initial collisions over LETZZ
The arrivals flow from NW are instructed to descend to FL80 over LETTZ and then descend to FL70, but this is the same also for collided flow from the SW. These two flows often collided over or passing LETZZ, prior switching to our control (only flights at or under FL70 are under our ATC control here) without possibility to solve it!
Thank you very much for the change in appropriate STARs for KDCA.
KDCA-collision

### LINKED ISSUE #1572: DCA Restricted Airspace
Browser: Chrome

Airport: DCA

Approximate play time before issue: Depends on Departure Sequence 

#### Issue Description
DCA departures on the JDUBB departure go through the Restricted airspace. This is after they have switched to center. I've tried issuing a climb to 11,000 but they don't climb above 7000. If I vector around the airspace, I get the error saying not on route. 

Screen Shot 2020-05-03 at 12 36 40 PM (2)

#### Steps to reproduce
- Happens every time a departure is headed through the restricted airspace. In the attached screen shot, the airplane RPA3193 is in the R-6608[a-c] restricted zones.

### PR DESCRIPTION
**Pull request by N864T**
Resolves #1538, resolves #1542, and resolves #1572 

The purpose of this pull request is to update KDCA and to (hopefully) resolve airspace-related issues.

### CODE REVIEW

**Comment by j-651** [comment]
Per @erikquinn, I've not added a changelog entry

File: documentation/airport-guides/kdca.md [hunk 1]
```diff
@@ -1,51 +1,78 @@
...
+The airport is known by many pilots for its River Visual procedure for runway 19 which requires at least 3 statute miles of visibility. It uses the Potomac River and various bridges including the Roosevelt Bridge, Memorial Bridge, Rochambeau Bridge and more.
 
-The area of the PCT which covers Richmond is the James River (JRV) area, which also covers the southern area of the PCT.
+As of <a href="https://www.mwaa.com/sites/default/files/12-19_ats_2.10.20.pdf" target="_blank">December 2019</a>, DCA has 579 daily departures via 17 airlines to 87 unique destinations.
 
-### Statistics
-Handled 70m+ passengers in 2015-2016
-800,000+ aircraft movements in 2015-2016
+### Airspace: Mount Vernon Area (MTV)
+The Mount Vernon Area is the part of the Potomac PCT which mainly covers DCA. The airspace is relatively large, so <a href="https://image.prntscr.com/image/fRaW9wieSJ_Ibi9KRix4Kg.png" target="_blank">here is a diagram</a>] of the airspace Mount Vernon owns.
```
Comment (Fechulo) [inline documentation/airport-guides/kdca.md:17]: There's an extra square bracket at the end of the hyperlink

File: documentation/airport-guides/kdca.md [hunk 2]
```diff
@@ -89,45 +119,66 @@ Times of use. Intermittent by NOTAM.
...
+|2/20|4,250|200|Asphalt|
 
 ### Videomap
 The videomap contains the following:
+
 * Restricted Airspaces;
-* Extended centreline for ILS01;
-* Various airports within the Mount Vernon Area;
-* Locations of important fixes
+* Extended centerlines for all runways (1L, 1C, 1R, 12, 19L, 19C, 19R, 30);
```
Comment (Fechulo) [inline documentation/airport-guides/kdca.md:173]: These are the IAD runways. The only extended centerline shown in the DCA map is the ILS for RWY1

File: documentation/airport-guides/kdca.md [hunk 3]
```diff
@@ -89,45 +119,66 @@ Times of use. Intermittent by NOTAM.
...
+* Various airports within in and around the Shenandoah Area;
```
Comment (Fechulo) [inline documentation/airport-guides/kdca.md:174]: This should probably be `Mount Vernon Area` or `Potomac TRACON`

**Comment by j-651** [comment]
Oops! Should be fixed now...

**Pull request merged**